atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/HikariCPConnectionManager.java

```diff
@@ -143,6 +146,24 @@ private void logPoolStats() {
     }
 
     private Connection getConnectionInternal() throws SQLException {
+        if (log.isInfoEnabled()) {
+            String host = dbConfig.getConnectionParameter(DBConfigConnectionParameter.HOST);
+            String ip;
+            try {
+                ip = InetAddress.getByName(host).getHostAddress();
```

Comment:
I realize this is info logging and we're trying to get specific details around DNS -> IP failover, but worth noting that if/when things go sideways, this DNS resolution may block. Nothing we can really do though